crates/telio-nurse/src/qos.rs

```diff
@@ -406,13 +407,27 @@ impl Analytics {
                     continue;
                 }
 
-                let (pk, endpoint) = (node.public_key, node.endpoint);
+                let (pk, mut ip_address) = (node.public_key, node.ip_address);
                 let pinger = Arc::clone(&self.ping_backend);
                 let ping_channel_tx = ping_channel_tx.clone();
 
                 tokio::spawn(async move {
                     if let Some(pinger) = &*pinger {
-                        Box::pin(pinger.perform((pk, endpoint), ping_channel_tx)).await;
+                        let mut dpr = Box::pin(pinger.perform((pk, ip_address))).await;
+
+                        // If we're pinging VPN address 100.64.0.1 and it's not
+                        // reachable, ping 10.5.0.1 instead
+                        if ip_address.target.0 == Some(Ipv4Addr::new(100, 64, 0, 1)) {
```

Comment:
It would be great if we moved these magic ip's into a singular place.
telio-model::contants maybe.

It would help tie code places. 
In genral would like that all peers could have multiple target's. and we would try them out in order.

Reply:
created telio-model/src/constants.rs to store these